spec/support/shared/spree_builders.rb

```diff
@@ -1,10 +1,14 @@
 RSpec.shared_context 'spree_builders' do
   def build_spree_product(id: 1, name: 'name', description: 'description',
+                          master: build_spree_variant,
+                          title: 'title',
                           vendor: 'vendor', slug: 'slug',
                           pos_product_id: '123',
                           created_at: build_date_time,
                           updated_at: build_date_time,
-                          available_on: build_date_time)
+                          available_on: build_date_time,
```

Comment:
Breaks, breaks, breaks! This is a lot of coupled data. Is there a better way to stub boundaries without recreating the whole data object as a double?